v2/cli/init_command.go

```diff
@@ -0,0 +1,248 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/urfave/cli/v2"
+)
+
+type InitFields struct {
+	ID       string
+	Language string
+	Git      string
+}
+
+var initCommand = &cli.Command{
+	Name:  "init",
+	Usage: "Initialize a template Brigade project",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    flagID,
+			Aliases: []string{"i"},
+			Usage: "Name of the Brigade project to create " +
+				"(required)",
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:    flagLanguage,
+			Aliases: []string{"l"},
+			Usage: "Set the language of the brigade project file" +
+				"to either JavaScript or TypeScript (default)",
+		},
+		&cli.StringFlag{
+			Name:    flagGit,
+			Aliases: []string{"g"},
+			Usage:   "Initialize the brigade project from an existing git repo",
```

Comment:
This usage text is a bit misleading. Try something like:

"Optionally specify a git repository where the event handling script can be found"